compiler/src/iree/compiler/Dialect/VMVX/Transforms/Passes.cpp

```diff
@@ -79,6 +79,8 @@ static void buildVectorVMVXTransformPassPipeline(OpPassManager &passManager) {
       createFixupSubspanWithOffsetsPass());
   nestedModulePM.addNestedPass<func::FuncOp>(
       createResolveBufferDescriptorsPass());
+  nestedModulePM.addNestedPass<func::FuncOp>(
+      createCleanupBufferAllocViewPass());
```

Comment:
what is this for?

Reply:
Some of the interface bindings might be dead, and this removes it, though with some recent changes we might not need this....